Extension/src/LanguageServer/ui.ts

```diff
@@ -104,6 +106,17 @@ export class OldUI implements UI {
         };
         this.ShowReferencesIcon = false;
 
+        this.compilerStatusItem = vscode.window.createStatusBarItem(`c.cpp.references.statusbar`, vscode.StatusBarAlignment.Right, 901);
+        this.compilerStatusItem.name = localize("c.cpp.compilerStatus.statusbar", "Compiler Status");
```

Comment:
Would "Configure IntelliSense" be more understandable to users since it would match the user text?